packages/camera/src/components/Controls/index.js

```diff
@@ -363,6 +363,11 @@ Controls.FullscreenButtonProps = {
   },
 };
 
+Controls.ExitFullscreenButtonProps = {
+  ...Controls.FullscreenButtonProps,
+  children: <FullScreenButton label="exitFullScreen" />,
+};
+
```

Comment:
instead of having 2 sets of props (`Controls.FullscreenButtonProps` and `Controls.ExitFullscreenButtonProps`) I think it would be better to have only one prop and pass a boolean parameter indicating if the app is in full screen mode